cli/src/command.rs

```diff
@@ -268,13 +268,27 @@ pub fn run() -> Result<()> {
 		Some(Subcommand::Benchmark(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
 			let chain_spec = &runner.config().chain_spec;
-
 			set_default_ss58_version(chain_spec);
-
-			Ok(runner.sync_run(|config| {
-				cmd.run::<service::kusama_runtime::Block, service::KusamaExecutor>(config)
-				.map_err(|e| Error::SubstrateCli(e))
-			})?)
+			let spec_name = runner.config().impl_name.to_lowercase();
+
+			if spec_name.contains("polkadot") {
```

Comment:
```suggestion
			if spec.is_polkadot() {
```

Please use this everywhere. There is the trait `IdentifyVariant` in `polkadot-service` that you will need to import for this.

Reply:
I would rather tweak the trait to be able to assert `spec.is_polkadot_dev()`, if you don't mind.

Reply:
`is_polkadot(Option<suffix>)`

Reply:
I was thinking of adding a `is_dev()` that return true on anything ending with `*-dev`.